Client/Frontend/Strings.swift

```diff
@@ -1071,6 +1071,69 @@ extension String {
                 comment: "In the settings menu, in the Tabs customization section, this is the title for the setting that toggles the Tab Groups feature - where tabs from related searches are grouped - on or off")
         }
 
+        public struct Notifications {
+            public static let Title = MZLocalizedString(
+                "Settings.Notifications.Title",
+                tableName: nil,
```

Comment:
These `tableName`s should not be nil. Moving forward, we'd like to make sure everything has a tableName. In this case, I'd suggest `"Settings"`